slynk/slynk-gray.lisp

```diff
@@ -99,6 +99,17 @@
 (defmethod stream-line-column ((stream sly-output-stream))
   (with-sly-output-stream stream column))
 
+(defmethod reset-stream-line-column ((stream sly-output-stream))
+  (with-sly-output-stream stream (setf column 0)))
+
+(defmethod reset-stream-line-column ((stream t))
+  nil)
+
+#+sbcl
+(defmethod reset-stream-line-column ((stream sb-sys:fd-stream))
```

Comment:
What happens on SBCL if we remove this one?

Reply:
I had to add this because SBCL doesn't seem to actually use a gray stream for output despite using slynk-gray.lisp. If you remove this one, then the T method discussed in the other comment is called and you get a REPL that still works, but not the alignment fix. Without either of these methods, I was getting a broken repl.

Reply:
OK, I'm starting to picture it.  So I'd say remove the other `T` method and keep this one.